client/src/components/exercise/lab11/pages/InformationLetterEmail.js

```diff
@@ -0,0 +1,124 @@
+import FogIndexOverlay from "../components/FogIndexOverlay";
+
+/**
+ * Renders the Information Letter Email page component.
+ * @param {Object} props - The component props.
+ * @param {Object} props.actions - The actions object.
+ * @param {string} props.words - The words string.
+ * @param {string} props.sentences - The sentences string.
+ * @param {string} props.complexWords - The complex words string.
+ * @param {boolean} props.isEditable - Whether the letter is editable.
+ * @param {Function} props.handleContinue - The function to handle continue button click.
+ * @returns {JSX.Element} The Information Letter Email page component.
+ */
+const InformationLetterEmail = (props) => {
+  const {
+    actions,
+    words,
+    sentences,
+    complexWords,
+    sectionTitle,
+    isEditable,
+    handleContinue,
+    descriptionText,
+  } = props;
+
+  useEffect(() => {
+    actions.updateState(EXERCISE_PLAYING);
+  }, []);
+
+  useScroll();
+
+  const {
+    letterContent,
+    totalWords,
+    setTotalWords,
+    totalSentences,
+    setTotalSentences,
+    totalComplexWords,
+    setTotalComplexWords,
+    fogIndex,
+    setFogIndex,
+  } = useContext(ExerciseStateContext);
+
+  useEffect(() => {
+    const { wordCount, sentenceCount, complexWordCount, fogIndex } =
+      fogIndexCalculation(letterContent, words, sentences, complexWords);
+    setTotalWords(wordCount);
+    setTotalSentences(sentenceCount);
+    setTotalComplexWords(complexWordCount);
+    setFogIndex(fogIndex);
+  }, [letterContent]);
+
+  return (
+    <div className="center-div">
+      {sectionTitle && <h1 className="playthrough__title">{sectionTitle}</h1>}
+      <div className="playthrough__sentence">{descriptionText}</div>
+
+      <div className="tw-flex flex-col tw-justify-center tw-relative">
+        <div className="tw-w-full tw-h-auto tw-bg-[#E8EBED] tw-rounded-2xl tw-shadow tw-py-5 tw-flex tw-flex-col">
+          <div className="tw-flex tw-justify-center">
+            <div className={`tw-w-[90%] tw-flex tw-flex-col`}>
+              <div
+                className={`tw-text-xl tw-self-start tw-my-4 tw-mb-5 tw-font-bold`}
+              >
+                New Message
+              </div>
+            </div>
+          </div>
+          <div className="tw-bg-white tw-flex tw-justify-center tw-pt-4">
+            <div className={`tw-w-[90%] tw-flex tw-flex-col `}>
+              <div className={`tw-opacity-50 tw-text-xl tw-self-start`}>
+                Recipents
+              </div>
+              <div className="tw-w-full tw-mx-auto tw-bg-[#B4B4B4] tw-h-[2px] tw-my-4 tw-opacity-40" />
+              <div className={`tw-opacity-50 tw-text-xl tw-self-start`}>
+                Subject
+              </div>
+              <div className="tw-w-full tw-mx-auto tw-bg-[#B4B4B4] tw-h-[2px] tw-my-4 tw-opacity-40" />
+              {/* Letter Content */}
+              <Letter isEditable={isEditable} />
```

Comment:
Editing the letter is broken -> when you start typing at the end of the letter, it moves your cursor to the beginning of the letter